platform/podjs/src/browserPod.ts

```diff
@@ -602,9 +702,10 @@ declare global {
 }
 
 /**
- * Returns the relative luminance value of a feature color.
- *
- * @param featureColor - A six digit hex color string, e.g. #000000
+ * It takes a string representing a color in hexadecimal format and
```

Comment:
Shouldn't the actual format of the parameter be specified on the `param` below - which it is?